.github/workflows/molecule_workflow.yml

```diff
@@ -0,0 +1,35 @@
+name: Role test workflow
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - "*"
+
+jobs:
+  build:
+    name: Test Roles using MOLECULE
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v3
+          
+      - name: Set up Python 
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.10'
```

Comment:
Is it possible to change this back to `3.x`? I don't think we need 3.10 specifically.